pom.xml

```diff
@@ -23,6 +23,9 @@
   <groupId>co.cask.cdap</groupId>
   <artifactId>cdap</artifactId>
   <version>5.0.0-SNAPSHOT</version>
+  <modules>
+    <module>cdap-runtime-monitor</module>
+  </modules>
```

Comment:
shouldn't this go under this list instead: https://github.com/caskdata/cdap/blob/976569470165605870fd2aa21588e0a1d51db28e/pom.xml#L2684